ckanext/switzerland/helpers/frontend_helpers.py

```diff
@@ -165,6 +165,8 @@ def get_dataset_by_identifier(identifier):
         )
     except logic.NotFound:
         return None
+    except Exception as e:
+        raise
```

Comment:
You don't need to catch an exception just to raise it again - that should happen anyway.